addons/sale_timesheet/data/sale_service_demo.xml

```diff
@@ -140,12 +159,12 @@
             <field name="order_id" ref="sale_timesheet.sale_order_2"/>
             <field name="sequence" eval="2"/>
             <field name="product_id" ref="product_service_deliver_timesheet_2"/>
-            <field name="product_uom_qty">10</field>
+            <field name="product_uom_qty">20</field>
         </record>
 
         <!-- Activity of sales order -->
         <record id="sale_timesheet_activity_1" model="mail.activity">
-            <field name="res_id" ref="sale_timesheet.sale_order_1"/>
+            <field name="res_id" ref="sale_timesheet.sale_order_2"/>
```

Comment:
If I remember correctly, it was easier to create and trigger an upsell threshold on the other SO, so I moved this activity so both SO would have one instead of two activities on the same SO